apps/search/src/components/AppSection.tsx

```diff
@@ -0,0 +1,36 @@
+import { Box, PropsWithBox, Text } from "@saleor/macaw-ui/next";
+import { ReactNode } from "react";
+
+// todo move to shared
+export const AppSection = ({
+  heading,
+  sideContent,
+  mainContent,
+  includePadding = false,
+  ...props
+}: PropsWithBox<{
+  heading: string;
+  sideContent?: ReactNode;
+  mainContent: ReactNode;
+  includePadding?: boolean;
+}>) => {
+  return (
+    <Box as="section" __gridTemplateColumns={"400px auto"} display={"grid"} gap={13} {...props}>
```

Comment:
I'm not sure if this will be flexible - I want left column fixed size, right column fluid.

your solution is single column on mobile (which we don't support) and 3 "automatic" columns on desktop (but I need 2).

So maybe I need yo check your solution first. 

Definitely we are going to make it standard component later